arduinolux/src/Debug/DebugTest.cpp

```diff
@@ -0,0 +1,14 @@
+#include"../StringConverter.h"
+#include <string>
+#include <iostream>
+
+int main() {
+	std::string testString = "Hello there world!";
+	do {
+		char* convertedString = StringConverter::ExtractCharArrayFromString(testString);
+
+		std::cout << convertedString << "\n";
```

Comment:
The nano has been replaced with a [Particle Photon](https://www.particle.io/products/hardware/photon-wifi-dev-kit).
we'd like to decouple the serialcontrol() method so that it can accept strings sent from the [Particle.io subscribe API](https://docs.particle.io/reference/firmware/photon/#particle-subscribe-)